_posts/2022-02-02-#748.md

```diff
@@ -10,19 +10,47 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- [libsecp256k1](https://github.com/bitcoin-core/secp256k1/) is the library that Bitcoin Core uses for low-level cryptographic operations such as signing and verification.
+
+- This PR adds usage examples for several common use cases: ECDSA signatures, Schnorr signatures, and (elliptic curve) Diffie-Hellman key exchange. There are no existing usage examples in the library.
+
+- The examples are intended to clarify the API by supplementing the documentation in the [various header files](https://github.com/bitcoin-core/secp256k1/tree/master/include).
+
 ## Questions
+
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
--->
+2. Can you compile and run the example (note: you must run `./configure` with the `--enable-examples` flag)?
```

Comment:
Is it worth mentioning that you also need `--enable-experimental --enable-ecdh --enable-schnorrsig` to get all the examples?

Reply:
Good point, added.